package.json

```diff
@@ -35,16 +35,20 @@
     "!src/stories"
   ],
   "scripts": {
-    "start": "storybook dev --port 6006",
-    "build": "npm run build:ts",
+    "start": "concurrently \"npm run storybook:start\" \"npm run css:start\"",
```

Comment:
> Side note on the CSS stuff: even without a bundler for `npm:build` at the moment, we still need to make sure it can integrate into our dev flow for storybook / the examples which do have proper tooling.

You're right, manually building css on changes was lame. Updated it here to auto build on css changes too:

https://user-images.githubusercontent.com/11449462/234885086-46895826-1293-4cb6-9542-f04bebdb1ec2.mp4